stable/wordpress/templates/deployment.yaml

```diff
@@ -13,6 +13,7 @@ spec:
     metadata:
       labels:
         app: {{ template "fullname" . }}
+        {{ template "mariadb.fullname" . }}-client: "true"
```

Comment:
What seems odd about that? We want the release name in the label so that one release of wordpress cannot talk to the database from another release of wordpress. The release name is essentially an ID.